packages/analytics/src/collect-analytics.ts

```diff
@@ -7,6 +7,8 @@ const interval = (configInterval || DEFAULT_INTERVAL_SECONDS) * 1000
 
 export default (): void => {
   setInterval(async () => {
+    const app = createApp()
```

Comment:
this will create a new app each time, this should be scoped outside of the setInterval